docs/streams/upgrade-guide.html

```diff
@@ -109,6 +109,23 @@
         Detailed metrics sensor can be found in the <a href="#kafka_streams_monitoring">Streams Monitoring</a> section.
     </p>
 
+    <p>
+        New methods in <code>KafkaStreams</code>:
+    </p>
+    <ul>
+        <li> retrieve the current runtime information about the local threads via <code>#localThreadsMetadata()</code> </li>
+    </ul>
+    <p>
+        Deprecated methods in <code>KafkaStreams</code>:
+    </p>
+    <ul>
+        <li><code>toString()</code></li>
+        <li><code>toString(final String indent)</code></li>
+    </ul>
+    <p>
+        The above methods have been deprecated in favor of using the new method <code>#localThreadsMetadata()</code>.
```

Comment:
We should also point to new class/method `TopologyDescription` and `Topology#describe()` here. Deprecated method `toStrint()` did give static and runtime information and it's functionality got split up in static information via `TopologyDescription` and runtime information provided via `ThreadMetadata`.

Reply:
Makes sense. @fhussonnois could you update the PR for this comment?